.github/workflows/ci.yaml

```diff
@@ -18,6 +18,9 @@ jobs:
 
         strategy:
             matrix:
+                config:
+                    - 'local'
+                    - 'rbe'
```

Comment:
These is one additional matrix case that would be good to test on CI which is the local but with `--experimental_allow_unresolved_symlinks` case. This should be the default at some point in the future as it will allow the link code to be simplified quite a bit.